packages/twenty-server/src/engine/core-modules/twenty-config/config-variables.ts

```diff
@@ -892,6 +892,16 @@ export class ConfigVariables {
   @IsOptional()
   SERVER_URL = 'http://localhost:3000';
 
+  @ConfigVariablesMetadata({
+    group: ConfigVariablesGroup.ServerConfig,
+    description: 'Base URL for public domains',
+    type: ConfigVariableType.STRING,
+    isEnvOnly: true,
```

Comment:
Why make it env only? I feel this could be configured at runtime without risk but might be wrong/missing something!